Artist show page #64

julsfelic opened the pull request:
Added the artist slug and styling.

Closes #64 

@scottfirestone @Draganovic @Salvi6God

julsfelic committed:
Add styling to artist page along with slug for url

Lots of styling refactoring.

julsfelic committed:
Fixes some styling bugs

Minor bug fixes with the updated styling.

scottfirestone commented:
@julsfelic   1) User can view an artist page they see all albums associated with artist

Aforementioned test is failing for me after migrating. Also check out this styling hiccup: 
![image](https://cloud.githubusercontent.com/assets/13153182/13556249/2f6c3f5e-e392-11e5-9895-76d0eff41018.png)

julsfelic committed:
Fix vertical centering bug with flexbox

Had to use flexbox instead of line height to fix the vertical align
of the h1. When we would have a long name the line break would
throw off the centering.

julsfelic committed:
Modifies test to comply with new styling

Had to slightly change the test because we are now delegating upcasing
the text to css and not ruby.

julsfelic commented:
Both the bug and spec are now fixed and passing.

@scottfirestone @Salvi6God @Draganovic

scottfirestone commented:
Looks good, thanks!
